cfgov/unprocessed/css/on-demand/pagination.less

```diff
@@ -0,0 +1,40 @@
+/* ==========================================================================
+   cfgov-refresh
+   Master Less File.
+   ========================================================================== */
+
+
+/* Base styles
+   @import combined CF components Less file and make any overrides.
+   ========================================================================== */
+
+// Import required CF components.
+// Combined CF components.
+@import (less) 'cf-core/src/cf-core.less';
+@import (less) 'cf-icons/src/cf-icons.less';
+@import (less) 'cf-buttons/src/cf-buttons.less';
+@import (reference) 'cf-layout/src/cf-layout.less';
+
+// The Capital Framework (CF) brand color palette.
+@import (reference) "../brand-palette.less";
+
+// Override the CF theme variables with colors from the brand color palette.
+@import (reference) "../cf-theme-overrides.less";
+
+// Enhancements that are on a migration path to being added to CF.
+// CF Core
+@import (reference) "../enhancements/vars.less";
+@import (reference) "../enhancements/media-queries.less";
+@import (less) "../enhancements/base.less";
+@import (less) "../enhancements/layout.less";
+
+
```

Comment:
I'm a bit worried about the number of on-demand components we're using. Each duplicates a lot of CF in individually delivered files increasing page load exponentially. Is there some way we can combine all the on-demand stuff? If you're loading the pagination, you're going to need the header/footer. If you're not loading the pagination, that little extra bit isn't going to even be noticeable compared to all the duplication we're creating. We shouldn't have to worry about polluting existing styles because we've prefixed everything with atomic type.